recipe/meta.yaml

```diff
@@ -12,7 +12,7 @@ source:
 
 build:
   number: 0
-  skip: True  # [py3k]
+  skip: True  # [osx and py3k]
```

Comment:
It looks like there is [trouble with Python 3.6](https://github.com/conda-forge/vtk-feedstock/pull/24) in one of the dependencies, maybe leave it out for now?